python/dgl/distributed/graph_partition_book.py

```diff
@@ -792,28 +845,19 @@ def __init__(self, part_id, num_parts, node_map, edge_map, ntypes, etypes):
         assert all(
             ntype is not None for ntype in self._ntypes
         ), "The node types have invalid IDs."
-        for etype, etype_id in etypes.items():
-            if isinstance(etype, tuple):
-                assert (
-                    len(etype) == 3
-                ), "Canonical etype should be in format of (str, str, str)."
-                c_etype = etype
-                etype = etype[1]
-                self._etypes[etype_id] = etype
-                self._canonical_etypes[etype_id] = c_etype
-                if etype in self._etype2canonical:
-                    # If one etype maps to multiple canonical etypes, empty
-                    # tuple is used to indicate such ambiguity casued by etype.
-                    # See more details in self._to_canonical_etype().
-                    self._etype2canonical[etype] = tuple()
-                else:
-                    self._etype2canonical[etype] = c_etype
+        for c_etype, etype_id in etypes.items():
+            assert isinstance(c_etype, tuple) and len(c_etype) == 3, \
+                f"Expect canonical edge type in a triplet of string, but got {c_etype}."
```

Comment:
over 80 chars.

Reply:
max-length 100?

Reply:
@Rhett-Ying No, max-length is 80.

Reply:
@frozenbugs Look like our linter is still using max-length=100. https://github.com/dmlc/dgl/blob/master/tests/lint/pylintrc#L340

Reply:
fixed in https://github.com/dmlc/dgl/pull/4809